<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrint_preview"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuFormat">
     <property name="title">
      <string>Format</string>
     </property>
     <addaction name="actionColour"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionFind_Replace"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="menuFormat"/>
   </widget>
   <widget class="QMenu" name="menuFormat_2">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionBold"/>
    <addaction name="actionItalix"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionStrikeout"/>
    <addaction name="separator"/>
    <addaction name="actionAlign_center"/>
    <addaction name="actionAlign_Left"/>
    <addaction name="actionAlign_right"/>
    <addaction name="actionAlign_justify"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionPrint_preview">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentPrintPreview"/>
   </property>
   <property name="text">
    <string>Print preview</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentPrint"/>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditUndo"/>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditRedo"/>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCut"/>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy"/>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditPaste"/>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
   <property name="text">
    <string>Find/Replace</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditSelectAll"/>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionColour">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/colour.png</normaloff>:/image/colour.png</iconset>
   </property>
   <property name="text">
    <string>Colour</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextBold"/>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalix">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextItalic"/>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextUnderline"/>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionStrikeout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextStrikethrough" resource="resource.qrc">
     <normaloff>:/image/text_strikethrough.png</normaloff>:/image/text_strikethrough.png</iconset>
   </property>
   <property name="text">
    <string>Strikeout</string>
   </property>
  </action>
  <action name="actionAlign_center">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatJustifyCenter"/>
   </property>
   <property name="text">
    <string>Align center</string>
   </property>
  </action>
  <action name="actionAlign_Left">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatJustifyLeft"/>
   </property>
   <property name="text">
    <string>Align Left</string>
   </property>
  </action>
  <action name="actionAlign_right">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatJustifyRight"/>
   </property>
   <property name="text">
    <string>Align right</string>
   </property>
  </action>
  <action name="actionAlign_justify">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatJustifyCenter"/>
   </property>
   <property name="text">
    <string>Align justify</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
